internal/pkg/aws/ecr/ecr.go

```diff
@@ -194,3 +201,14 @@ func URIFromARN(repositoryARN string) (string, error) {
 		repoARN.Region,
 		repoName), nil
 }
+
+func isRepoNotFoundErr(err error) (bool, error) {
+	aerr, ok := err.(awserr.Error)
+	if !ok {
+		return false, err
+	}
+	if aerr.Code() == "RepositoryNotFoundException" {
+		return true, nil
+	}
+	return false, err
+}
```

Comment:
I believe we can remove the returned error from this function's signature:
```go
func isRepoNotFoundErr(err error) bool {
	aerr, ok := err.(awserr.Error)
	if !ok {
		return false
	} 
	if aerr.Code() == "RepositoryNotFoundException" {
		return true
	}
	return false
}
```
If the err is not an awserr, it's by definition not a RepoNotFound error.